### ISSUE TITLE
task02: irises

### ISSUE THREAD

**Issue by Revilllo7** [Revilllo7/computational-intelligence-class#47]
Utilise Pytorch and the tutorial to create a python script that takes data from isis_big.csv, trains and validates the neural network.

Do the following:
- Use the dedicated tools for data loading and it's edition `DataLoading` and (normalisation/data edition)
- Create a neural network structure that makes sense (explain why it makes sense structure-wise: topology, activation functions, etc)
- Create a training loop in Pytorch
- Give the final validation statistics: accuracy, confusion matrix for the entire validation dataset
- interpret the results
